Brokerages/GDAX/GDAXBrokerage.cs

```diff
@@ -77,16 +76,16 @@ public override bool PlaceOrder(Order order)
 
                 if (raw?.Id == null)
                 {
-                    OnMessage(new BrokerageMessageEvent(BrokerageMessageType.Error, (int)response.StatusCode, "GDAXBrokerage.PlaceOrder: Error parsing response from place order: " + response.Content));
+                    OnMessage(new BrokerageMessageEvent(BrokerageMessageType.Warning, (int)response.StatusCode, "GDAXBrokerage.PlaceOrder: Error parsing response from place order: " + response.Content));
                     UnlockStream();
-                    return false;
+                    return true;
```

Comment:
Should we be returning true here? I think we should still be returning false so the transaction handle invalidates the QC order.
![image](https://user-images.githubusercontent.com/531416/34059625-bbfb8838-e1ad-11e7-9284-87c49b010dc2.png)

Reply:
Maybe not this exact case since we couldn't get an ID, but the 'rejected' case below should certainly return `false`, yes/no?

Reply:
Hmm, if we return `false` in these cases, the order status will be correctly set to `Invalid`, but the algorithm will also be terminated, due to the `_algorithm.Error` call above -- which is the issue we are trying to solve in this PR :)
Looking better at the code, I should probably still return `true` but add a call this line in all error cases (like in the last error case at the end of the method) ?
```
 OnOrderEvent(new OrderEvent(order, DateTime.UtcNow, 0, "GDAX Order Event") { Status = OrderStatus.Invalid });
```
This line also updates the order status